instance/management/commands/instance_statistics_csv.py

```diff
@@ -161,6 +161,7 @@ def get_instance_metadata(self, instances):
             name_prefix = appserver.server_name_prefix
             domain = instance.domain
             ref_name = instance.ref.name
+            external_lms_domain = instance.external_lms_domain
```

Comment:
@alfredchavez, instead of using the `''` which is the default value for this model field, can you use something like `'N/A'` like it is done for the other fields?